deploy/job-manager/README.md

```diff
@@ -0,0 +1,36 @@
+# Job Manager
+
+This folder hosts the config files and script for deploying the Job Manager on GreenBox's Cromwell instance. 
+
+## Available Environments
+- dev
+- staging
+- integration
+- prod: _No production deployments yet_
+
+## Deployment Process
+
+To deploy the Job Manager:
+0. Make sure you have created the ingress for the job-manager in advance, refer to `job-manager-ingress.yaml.ctmpl` for the content of the ingress YAML file.
```

Comment:
This line is not getting indented properly...does it need to start from `1.`?

Reply:
I think it's useful to include the kubectl command for creating the ingress here `kubectl apply -f job-manager-ingress.yaml`